### ISSUE TITLE
No rspec tests

### ISSUE THREAD

**Issue by patrickbrown-dev** [rubynoobs/beaker#5]
I've embarassingly not implemented a single rspec test in beaker. However, that does not negate the fact that we need to practice writing rspec tests.

I think we might need a ruby on rails mentor for this one?

**Comment by n8fischer** [comment]
indeed... perhaps qpewq can help us out there

**Comment by Jacobichou** [comment]
Nah! Not just yet! I've gotten fairly good with tests! I know I keep saying this, but I'm taking a look today. Work keeps exploding on me. Today will be fine though, for sure.
